### ISSUE TITLE
Not working on WebStrom 2017.3

### ISSUE THREAD

**Issue by TomazWang** [wallabyjs/quokka#155]
### Issue description or question
Nothing happened when use Quokka after I installed Quokka and follow the steps in the tutorial. Nothing shows up in my editor nor in the Quokka console. 

### Sample code

```javascript
const greet = () => {
  console.log('hi');
};

greet();
```

### Quokka.js Console Output

```
Quokka #1 (node: v8.8.1)

// nothing here... 
```

### Code editor version

Webstorm v2017.3.4

### OS name and version

OSX 10.13.3

### Other information 

I've taken a snapshot of my IDE.

![](https://user-images.githubusercontent.com/7992586/35934760-6b4f1cf0-0c79-11e8-913f-7dcb5773caad.png)

And here's my `~/.quokka/config.json` file:

```json
{
  "node": "/usr/local/bin/node",
  "pro": false
}
```

I notice that there was no time shows up next to the little x sign. The console sometimes blinks after I enter new line (like it restarts after it crashes), however, I couldn't find any logs in my Webstorm log file.

**Comment by jose4125** [comment]
I installed the plugin on Webstrom 2017.3.4 (OSX 10.12.6) a few minutes ago and it doesn't work, happen the same issue than @TomazWang

**Comment by ArtemGovorov** [comment]
The issue is fixed and the fix is published in v1.0.68.
